app/src/main/java/com/kodeco/android/coordplot/MainActivity.kt

```diff
@@ -46,10 +72,19 @@ fun PlotSurfacePreview() {
 
 @Composable
 fun Map(xPercent: Float, yPercent: Float, modifier: Modifier = Modifier) {
-    // TODO fill out the square map here.
-    //  Create a Box composable with a size of 300.dp
-    //  and then create an inner Box composable
-    //  with a clip shape of CircleShape.
+    Box(
+        modifier
+            .size(300.dp)
```

Comment:
Maybe introduce a constant for the value 300?

Reply:
I can store all dimensions of the UI to the constant file

Reply:
@Teqx  
I added constants